### PR TITLE
Add input-duration component to rollup

### PR DESCRIPTION
**Pull request by jorge-ramirez-arredondo**
[US127919](https://rally1.rallydev.com/#/431372673576d/iterationstatus?detail=%2Fuserstory%2F603444721280&fdp=true?fdp=true)

https://github.com/BrightspaceUILabs/input-duration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
   },
   "homepage": "https://github.com/Brightspace/brightspace-integration",
   "dependencies": {
+    "@brightspace-ui-labs/input-duration": "^1.1.1",
```
Comment (dlockhart) [inline package.json:32]: We typically don't bother specifying the minor/patch release just to keep things clean. Then the lock file is where the nitty gritty is tracked anyway.

```suggestion
    "@brightspace-ui-labs/input-duration": "^1",
```

**Review by dlockhart — APPROVED** [review]
Looks good. I kicked the build as it failed on a pesky memory problem we've been seeing more and more.

**Review by dlockhart — APPROVED** [review]

**Comment by dlockhart** [comment]
I was able to get this going by increasing the `--max-old-space-size` flag via `NODE_OPTIONS` environment variable in Travis CI. Disaster averted for now I think.

**Comment by jorge-ramirez-arredondo** [comment]
> I was able to get this going by increasing the `--max-old-space-size` flag via `NODE_OPTIONS` environment variable in Travis CI. Disaster averted for now I think.

Is this going to continue to cause problems if more and more packages are added to BSI?

**Comment by dlockhart** [comment]
> Is this going to continue to cause problems if more and more packages are added to BSI?

I don't _think_ so. The way I understand that flag is that it's telling the garbage collection to get more aggressive as it reaches that value. So if we've already reached it here with this PR and this flag solved it, then in theory adding more and more shouldn't matter. But who knows!

Stacey and I will be starting a project shortly that will try to find better ways to serve up our new component-based apps than BSI. It's problematic for several reasons, this memory thing just being the latest.

**Pull request merged**